src/sbl_filing_api/entities/repos/submission_repo.py

```diff
@@ -156,18 +156,10 @@ async def update_contact_info(
 
 async def add_user_action(
     session: AsyncSession,
-    user_id: int,
-    user_name: str,
-    user_email: str,
-    action_type: str,
+    new_user_action: UserActionDTO,
 ) -> UserActionDAO:
-    new_user_Action = UserActionDAO(
-        user_id=user_id,
-        user_name=user_name,
-        user_email=user_email,
-        action_type=action_type,
-    )
-    return await upsert_helper(session, new_user_Action, UserActionDAO)
+    new_user_action = UserActionDAO(**new_user_action.__dict__.copy())
```

Comment:
I think you can just pass the dto to the helper function, right? it's already doing the conversion in the helper function from what I can see.

Reply:
Oh yeah, the helper function takes dot as well.